dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java

```diff
@@ -39,13 +42,30 @@ public String launchInstructions(String trsID) {
     @Override
     public VersionTypeValidation validateWorkflowSet(String initialPath, String contents,
         Map<String, Pair<String, GenericFileType>> indexedFiles) {
-        VersionTypeValidation validation = new VersionTypeValidation(true, new HashMap<>());
-        //        for(String line : contents.split("\\r?\\n")) {
-        //            if (!line.startsWith("import") && !line.startsWith("author") && !line.startsWith("description")) {
-        //                validation.setValid(false);
-        //                validation.getMessage().put(initialPath, "unknown keyword");
-        //            }
-        //        }
+
+        Map<String, String> validationMessageObject = new HashMap<>();
+
+        Yaml yaml = new Yaml();
+        List<String> files;
+        boolean isValid = true;
+        try {
+            Map<String, Object> map = yaml.load(contents);
+            Map<String, Object> serviceObject = (Map<String, Object>)map.get("service");
+            if (serviceObject == null) {
+                validationMessageObject.put(initialPath, "Service key does not exist.");
```

Comment:
Suggest the `The key "service" does not exist`, otherwise the error message might be read as there should be a capital S.